code/UI/WindInfoPanel.cs

```diff
@@ -1,26 +1,29 @@
 ﻿using Sandbox.UI;
 using Sandbox.UI.Construct;
+using System;
+using TerryForm.States.SubStates;
 
 namespace TerryForm.UI
 {
 	public class WindInfoPanel : Panel
 	{
+		public static WindInfoPanel Instance { get; set; }
```

Comment:
I know this is picky, but there should really be an empty line between this property and the constructor.